src/api/c/quiz3/main.c

```diff
@@ -6,106 +6,122 @@
 #include <unistd.h>
 #include <stdlib.h>
 #include <string.h>
+#include <math.h>
 
 #ifndef HEXDUMP_COLS
 #define HEXDUMP_COLS 8
 #endif
 
 #define DEVICE_NAME "/dev/ISSIMAGEDRV"
+#define FILE_NAME_1 "/boot/initramfs-4.14.0-1.el6.elrepo.x86_64.img"
```

Comment:
Better try something that is more guaranteed to be present on the file system, like own `Makefile` or `main.c`...